src/model_theory/fraisse.lean

```diff
@@ -253,9 +253,9 @@ variables {L} (K)
 
 /-- A structure `M` is a Fraïssé limit for a class `K` if it is countably generated,
 ultrahomogeneous, and has age `K`. -/
-structure is_fraisse_limit [countable_functions L] : Prop :=
+structure is_fraisse_limit [countable (Σl, L.functions l)] : Prop :=
 (ultrahomogeneous : is_ultrahomogeneous L M)
-(countable : (univ : set M).countable)
+(is_countable : (univ : set M).countable)
```

Comment:
Why not use `_root_.countable M` here?

Reply:
Hopefully I've caught all the uses of outdated definitions of countability now.